setup.py

```diff
@@ -100,7 +100,7 @@
             "Topic :: Software Development",
         ],
         python_requires=">=3.5",
-        install_requires=["deprecated", "pyjwt", "requests>=2.14.0,<2.25"],
+        install_requires=["deprecated", "pyjwt", "requests>=2.14.0"],
         extras_require={"integrations": ["cryptography"]},
         tests_require=["cryptography", "httpretty>=0.9.6"],
```

Comment:
This doesn't match test-requirements

Reply:
I hadn't realized the test requirements were duplicated in two places. Should be fixed by 91098f6